pombola/south_africa/management/commands/south_africa_import_new_constituency_office_locations.py

```diff
@@ -0,0 +1,138 @@
+"""Command to import PMG's constitiency office data
+
+PMG are undertaking an exercise to visit all the constituency
+offices and collect accurate locations and photographs.
+The data from this is available at
+
+https://app.m4jam.com/app/campaigns/2298/export/
+
+This script parses the resulting CSV file and updates our data
+on constituency offices to have the correct location.
+"""
+
+
+import csv
+import os.path
+import re
+
+import requests
+
+from django.conf import settings
+
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.gis.geos import Point
+
+from pombola.core.models import (
+    Organisation,
+    )
+from pombola.core.utils import mkdir_p
+
+from pombola.images.models import Image
+
+
+# Matches the format of the locations in the CSV we get from PMG.
+location_re = re.compile(
+    r'SRID=4326;POINT\(-?([0-9]+\.[0-9]+) (-?[0-9]+\.[0-9]+)\)')
+
+
+class Command(BaseCommand):
+    args = '<path to csv file>'
+    help = 'Updates constituency offices based on the supplied CSV file.'
+
+    def handle(self, *args, **options):
+        if len(args) != 1:
+            raise CommandError(
+                "You must provide the path to the CSV file as an argument."
+                )
+
+        constituency_offices = Organisation.objects.filter(
+            kind__slug='constituency-office')
+
+        desired_storage_path = os.path.join('images', 'constituency-offices')
+        images_directory = os.path.join(
+            settings.MEDIA_ROOT,
+            desired_storage_path
+            )
+        mkdir_p(images_directory)
+
+        with open(args[0]) as csvfile:
+            reader = csv.DictReader(csvfile)
+
+            for row in reader:
+                party, name = (
+                    re.match(
+                        '(.*) Constituency Office (.*)', row['POI_REFERENCE']
+                        ).groups()
+                    )
+                name = re.sub('\s+', ' ', name)
+
+                qs = constituency_offices.filter(
+                    name__regex=r'^{}.+: {}$'.format(party, name))
+
+                try:
+                    org = qs.get()
+                except Organisation.MultipleObjectsReturned:
+                    print "{} Orgs returned: {}".format(
+                        qs.count(),
+                        repr(qs),
+                        )
```

Comment:
I think the script should exit in this case - at the moment it'll struggle on but with the old value for `org`